Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Utils/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Utilities.asmdef

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Handtracking.Utilities",
+    "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Utilities",
```

Comment:
Do you know if there's a reason this assembly exists? It seems to contain one script (HandPoseUtils.cs), and the other script we use from MRTK-Quest (OculusHand.cs) is in the root `Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus` assembly. Can we remove this asmdef?

Reply:
I think we had it sequestered off initially because it used some Oculus package specific code before, removing this asmdef once ensuring its no longer needed.